images/dind/master/ovn-kubernetes-master.sh

```diff
@@ -15,7 +15,7 @@ function ovn-kubernetes-master() {
 
   local master_config="${config_dir}/master-config.yaml"
   cluster_cidr=$(python -c "import yaml; stream = file('${master_config}', 'r'); y = yaml.load(stream); print y['networkConfig']['clusterNetworks'][0]['cidr']")
-  apiserver=$(oc --config="${kube_config}" config view -o custom-columns=server:clusters[0].cluster.server | grep http)
+  apiserver=$(grep server ${kube_config} | cut -f 6 -d' ')
```

Comment:
Adding `-m1`, adding a colon to the pattern, and using `awk` would make the command more robust:
```bash
apiserver=$(grep -m1 server: /etc/origin/master/admin.kubeconfig | awk '{print $2}')
```

Reply:
@Miciah fixed, thanks.